pynamodb/models.py

```diff
@@ -525,6 +525,8 @@ def count(cls,
         :param filters: A dictionary of filters to be used in the query
```

Comment:
Can we also make it clear here (or on the first comment line) that `hash_key` is required?